docs/NMOS With H.264.md

```diff
@@ -58,11 +58,116 @@ The term 'strict-CBR' corresponds to the definition of constant bit-rate at sect
 The terms CBR and VBR are defined in the [ITU-T H.264][H.264] | ISO/IEC 14496 standard.
 
 ## H.264 IS-04 Sources, Flows and Senders
+
+Nodes capable of transmitting H.264 video streams MUST have Source, Flow and Sender resources in the IS-04 Node API.
+
 ### Sources
+
+The Source resource MUST indicate `urn:x-nmos:format:video` for the `format` attribute. 
+- Source resources MAY be associated with many Flows at the same time. The Source is therefore unaffected by the use of H.264 compression.
+
 ### Flows
+
+The Flow resource MUST indicate `video/H264` in the `media_type` attribute, and `urn:x-nmos:format:video` for the `format` attribute. This has been permitted since IS-04 v1.1. 
+- H.264 Flow resources MAY be associated with many Senders at the same time through the Senders' `flow_id` attributes. The H.264 Flow is therefore unaffected by the use of a specific transport. 
+- H.264 Flow resources MAY be associated with many Flows at the same time through the Flows' `parents` attributes. The H.264 Flow is therefore unaffected by being the parent of some other Flows.
+
+For Nodes implementing IS-04 v1.3 or higher, the following additional requirements on the Flow resource apply.
+
+In addition to those attributes defined in IS-04 for all coded video Flows, the following attributes defined in the [Flow Attributes Register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/) of the [NMOS Parameter Registers][] are used for H.264.
+
+These attributes provide information for Controllers and Users to evaluate stream compatibility between Senders and Receivers.
+
+- [Components](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#components)
+  The Flow resource MUST indicate the color (sub-)sampling, width, height and depth of the associated uncompressed picture using the `components` attribute. The `components` array values MUST correspond to the stream's active parameter sets values. A Flow MUST track the stream's current active parameter sets.
```

Comment:
```suggestion
  The Flow resource MUST indicate the color (sub-)sampling, width, height and depth of the associated uncompressed picture using the `components` attribute. The `components` array values MUST correspond to the stream's active parameter sets. A Flow MUST track the stream's current active parameter sets.
```

(or "values in the stream's active parameter sets", to avoid having to have another apostrophe like "stream's active parameter sets' values")

Reply:
done